terraform/prometheus.tf

```diff
@@ -28,7 +28,75 @@ resource "helm_release" "kube-prometheus-stack" {
     value = "default"
   }
 
-  depends_on = [ scaleway_k8s_pool.default, local_file.kubeconfig]
+  # Add Jitsi-Meet dashboards to Grafana
+  # Dashboards are fetched from https://github.com/systemli/prometheus-jitsi-meet-exporter/tree/1.2.1/dashboards
+  # Values are merged with https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml
+  set {
+    name = "grafana.dashboardProviders.dashboardproviders\\.yaml.apiVersion"
+    value = 1
+  }
+
+  set {
+    name = "grafana.dashboardProviders.dashboardproviders\\.yaml.providers[0].name"
+    value = "jitsi"
+  }
+
+  set {
+    name = "grafana.dashboardProviders.dashboardproviders\\.yaml.providers[0].orgId"
+    value = 1
+  }
+
+  set {
+    name = "grafana.dashboardProviders.dashboardproviders\\.yaml.providers[0].options.path"
+    value = "/var/lib/grafana/dashboards/jitsi"
+  }
+
+  set {
+    name = "grafana.dashboards.jitsi.jitsi-meet.url"
+    value = "https://raw.githubusercontent.com/systemli/prometheus-jitsi-meet-exporter/1.2.1/dashboards/jitsi-meet.json"
+  }
+
+  set {
+    name = "grafana.dashboards.jitsi.jitsi-meet.datasource"
+    value = "Prometheus"
+  }
+
+  set {
+    name = "grafana.dashboards.jitsi.jitsi-meet-system.url"
+    value = "https://raw.githubusercontent.com/systemli/prometheus-jitsi-meet-exporter/1.2.1/dashboards/jitsi-meet-system.json"
+  }
+
+  set {
+    name = "grafana.ingress.enabled"
+    value = lookup(var.k8s_grafana_ingress_enabled, terraform.workspace, false)
+  }
+
+  set {
+    name = "grafana.ingress.ingressClassName"
+    value = "nginx"
+  }
+
+  set {
+    name = "grafana.ingress.hosts[0]"
+    value = lookup(var.k8s_grafana_ingress_host, terraform.workspace, "monitoring.example.com")
+  }
+
+  set {
+    name = "grafana.ingress.annotations.cert-manager\\.io/issuer"
+    value = "letsencrypt-jitsi-issuer"
+  }
+
+  set_sensitive {
+    name = "grafana.adminPassword"
+    value = lookup(var.k8s_grafana_admin_passord, terraform.workspace, "ThisIsNotASecurePassword")
```

Comment:
better require setting the password than setting an unsafe password and risk shipping it to production?

Reply:
this parameter is mandatory in the `lookup` function